.travis.yml

```diff
@@ -27,9 +27,6 @@ env:
                         OPENBLAS_NUM_THREADS=2
                         SKLEARN_BUILD_PARALLEL=8
                         SKLEARN_SKIP_NETWORK_TESTS=1"
-    # Nightly upload token and staging upload token are set in Travis settings
-    - SCIKIT_LEARN_NIGHTLY_UPLOAD_TOKEN=__token__
-    - SCIKIT_LEARN_STAGING_UPLOAD_TOKEN=__token__
```

Comment:
I do not think we set these variables yet in https://travis-ci.com/github/scikit-learn/scikit-learn/settings

I think @ogrisel has access to these tokens.

Reply:
The point is that these environment variables are unnecessary since they will be set through `Travis` secrets.

This PR just removes these two unnecessary lines (maybe the PR title does not properly describe the purpose of the PR).

Reply:
Yes, removing them makes sense to do.